### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by sastels**
# Description

run `npm audit fix` and then update patch and minor versions

# Any new packages installed?

no new packages, just updated lots. Will require everyone to do a `npm install`

# Required followup work

Still have major version updates to do, some of which will require code refactoring.
- nanoid
- @storybook/preset-create-react-app
- @testing-library/react
- cypress-file-upload
- mochawesome
- @chakra-ui/core

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code
- [x] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

File: f2/src/forms/__tests__/ContactInfoForm.test.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ describe('<ContactInfoForm />', () => {
       </MemoryRouter>,
     )
 
-    const inputNode = getAllByRole('textbox')[0]
+    const inputNode = getAllByLabelText('contactinfoPage.fullName')[0]
```
Comment (sastels) [inline f2/src/forms/__tests__/ContactInfoForm.test.js]: the `getAllByRole` started complaining that it couldn't find a `textbox`.

File: f2/package.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@azure/storage-blob": "^12.1.0",
+    "@azure/storage-blob": "^12.1.1",
     "@chakra-ui/core": "^0.5.2",
```
Comment (amazingphilippe) [inline f2/package.json]: I saw there are newer versions of chakra? are they breaking changes?

File: f2/src/forms/__tests__/ContactInfoForm.test.js (hunk 1 above)
Comment (amazingphilippe) [reply f2/src/forms/__tests__/ContactInfoForm.test.js]: We don't have textboxes on contact info.... 🤔

Comment (sastels) [reply f2/src/forms/__tests__/ContactInfoForm.test.js]: 🤷‍♂

**Comment by sastels** [comment]
Snyk not running because CDS only has the free plan 🤦‍♂

**Pull request merged**